packages/ui-components/cypress/support/customPercyCommand.ts

```diff
@@ -54,6 +55,54 @@ declare global {
   }
 }
 
+function setupAxeAndCheckA11y (isComponentTesting?: boolean) {
+  if (isComponentTesting) {
+    Cypress.Commands.add('injectAxe', () => {
+      // this is a work around for the issue with require.resolve in CT
+      // described here: https://github.com/component-driven/cypress-axe/issues/134
+      cy.window({ log: false }).then((window) => {
```

Comment:
Small insignificant nit! It would be nice if we could dynamically determine `isComponentTesting` instead of passing it around. We coullllldddddd do something like the following. I'm not familiar with the danger zones of this approach tho so take this with a grain of salt 🧂 

```
function setupAxeAndCheckA11y () {
  cy.window({ log: false }).then((window) => {
    const isComponentTesting = window.__CYPRESS_TESTING_TYPE__ === 'component'
    // rest of logic as you had it
  })
}
```

Reply:
Totally. The only reason not to do this is my pretty slight personal preference to avoid those global variables unless they are absolutely necessary. They are mostly there because in run mode we don't use GQL, so there are some things that typically would be read out of GQL, that get set on the window to help the Spec Runner initialize correctly in run mode.

Since the support files know what the testing type is, I figure we don't need to bind this to the global, and the function can just know what to do when it's called.

That said, who knows what a more fleshed out implementation of this will look like, might not even be in the same command, so I also see this as pretty temporary.

Reply:
Better yet, you can just do `Cypress.testingType`: https://docs.cypress.io/api/cypress-api/testing-type

How about using that?

Reply:
one. `Cypress.testingType` feels like something that's supposed to be there. I briefly got interested in whether we can replace some of the other `__CYPRESS_TESTING_TYPE__` usages, but it wasn't an instant win so if there are some gains there we can do it in another PR.